internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -917,7 +917,6 @@ components:
         - tenant_id
         - subscription_id
         - resource_group
-        - location
```

Comment:
Can you document on the line 943 what happens when the location isn't defined? (The location is determined from the RG.)

Reply:
sure

Reply:
done